frontend/public/components/monitoring/silence-form.tsx

```diff
@@ -419,4 +419,4 @@ type SilenceFormProps = {
   defaults: any;
   Info?: React.ComponentType<{}>;
   title: string;
-};
+};
```

Comment:
Unintentional change?

Reply:
When the codes were copied&pasted from  #7386, the newline was missed at the end of file.
After add the newline  at the end of file, it will pass prettier/prettier.

Reply:
Done